reactnative/ReactNativeTemplateApp/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "ReactNativeTemplateApp",
+  "name": "ReactNativeTemplate",
```

Comment:
Should this name match the declaration in ```MainActivity``` below, i.e., ```ReactNativeTemplateApp```? I'm not sure how the flow works here.